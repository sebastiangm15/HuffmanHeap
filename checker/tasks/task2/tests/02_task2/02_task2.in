8
5 SAT1
5 SAT2
7 SAT3
7 SAT4
10 SAT5
10 SAT6
12 SAT7
15 SAT8
4
0001
0010
0100
1011
