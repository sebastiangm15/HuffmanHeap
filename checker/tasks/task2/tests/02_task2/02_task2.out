SAT3 
SAT4 
SAT8 
SAT5 
